scripts/release/debian/deb_install.sh

```diff
@@ -42,12 +42,13 @@ setup() {
     set -v
     export DEBIAN_FRONTEND=noninteractive
     apt-get update
-    apt-get install -y apt-transport-https lsb-release gnupg curl
+    apt-get install --assume-yes --no-install-recommends apt-transport-https ca-certificates curl gnupg2 lsb-release
     set +v
 
     assert_consent "Add Microsoft as a trusted package signer?" ${global_consent}
     set -v
-    curl -sL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /etc/apt/trusted.gpg.d/microsoft.gpg
+    curl -fssL https://packages.microsoft.com/keys/microsoft.asc |\
```

Comment:
```suggestion
    sudo mkdir -p /etc/apt/keyrings
    curl -fssL https://packages.microsoft.com/keys/microsoft.asc |\
```
`/etc/apt/keyrings` does not exists in Ubuntu 20.04 and 18.04.

Reply:
nice catch @bebound, I was focusing on 24.04